Vagrantfile

```diff
@@ -6,17 +6,24 @@ NODES = 4
 DISKS = 8
 
 Vagrant.configure("2") do |config|
-    config.vm.box = "chef/centos-7.1"
```

Comment:
Let me check that and get back to you. I didn't test the vagrant-mutate'd
box without skipping setting up of enp0s8 interface.
On 02-Jul-2015 5:58 am, "Luis Pabón" <notifications@github.com> wrote:

> In Vagrantfile
> <https://github.com/heketi/vagrant-heketi/pull/5#discussion_r33738462>:
>
> > @@ -6,17 +6,24 @@ NODES = 4
> >  DISKS = 8
> >
> >  Vagrant.configure("2") do |config|
> > -    config.vm.box = "chef/centos-7.1"
>
> so do you still need to make changes to the ansible script to make the
> correct configuration inside the VM?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/heketi/vagrant-heketi/pull/5/files#r33738462>.
>